src/Components/js/Cards.jsx

```diff
@@ -0,0 +1,45 @@
+function Cards() {
+  return (
+    <>
```

Comment:
Fragments are representations of divs and are used in React when you need to wrap independent children under one parent. But since you have a container div, you don't need an extra div at the top